### PR TITLE
Update HasPermissionsTest.php

### PR DESCRIPTION
**Pull request by musapinar**
Fixed random test failure adding ```->sort()->values()``` to ```pluck()```
Exact same issue as https://github.com/spatie/laravel-permission/pull/1324

### CODE REVIEW

**Comment by drbyte** [comment]
I've never seen these "random test failures" that you mention, neither in local dev nor in CI.
What's causing it for you?

**Comment by musapinar** [comment]
> 
> 
> I've never seen these "random test failures" that you mention, neither in local dev nor in CI.
> What's causing it for you?

As you can see in the screenshot here: https://i.imgur.com/MoS3erH.png
The Collection's key/value pair may be different.

Sometimes I have :
```
Illuminate\Support\Collection {#1638
  #items: array:2 [
    0 => "Edit Articles"
    1 => "Edit News"
  ]
}
```

Some other times I have : 
```
Illuminate\Support\Collection {#1638
  #items: array:2 [
    0 => "Edit News"
    1 => "Edit Articles"
  ]
}
```
Which causes the subsequent assertion to fail randomly.
So I am forcing alphabetical order with ```->sort()->values()```

Context:
Laravel 6.10
PHP 7.3

After further investigations, I figured out it's actually because I am using UUIDs instead of this package's default bigIncrement IDs. ```pluck()``` orders by ID ASC indeed by default, but in my case the UUIDs are generated on the fly via a model observer so it makes no sense to sort by UUID and explains perfectly why it randomly failed.

TL;DR : False flag. Sorry about that.

**Comment by drbyte** [comment]
For consistency with the other PR already merged, I'll merge this too. 
While there are no plans to natively force UUIDs in this package, this is not a problematic change.
